src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx

```diff
@@ -14,8 +14,8 @@ This tutorial will walk you through adding and managing accounts and users.
 ## Before you start [#requirements]
 
 Before you start this tutorial, some things to understand: 
-* Pro or Enterprise edition is required (Standard edition can't add accounts or create groups or roles)   
-* This tutorial is for managing users who are on our [newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models). 
+* Pro or Enterprise [edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#editions) is required
+* This tutorial is for managing users who are on our [newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) (true of almost all New Relic users)
```

Comment:
this was my strategy for making it as simple as possible, because it was thought we would not have another user model for a long time. there are few references to the user model distinction and hopefully soon we can entirely get rid of the mentions of all that stuff because there'll only be a single user model.